### PR TITLE
Fix copy in email templates

### LINKED ISSUE #2784: Password reset email copy
The current password reset template used for the emails have this copy:
`Please click the following link to verify your email address within 3 hours:`

Which to me is misleading. Something like this:
`Please click the following link to reset your password within 3 hours:`

### PR DESCRIPTION
**Pull request by mmarif4u**
Fixes https://github.com/go-gitea/gitea/issues/2784

### CODE REVIEW

File: templates/mail/auth/activate_email.tmpl [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-	<title>{{.Username}}, please verify your e-mail address</title>
+	<title>{{.Username}}, please verify your email address</title>
```
Comment (lafriks) [inline templates/mail/auth/activate_email.tmpl]: I don't think this change is needed

Comment (lafriks) [reply templates/mail/auth/activate_email.tmpl]: I think it would be better as it was before

Comment (mmarif4u) [reply templates/mail/auth/activate_email.tmpl]: Ok, no problem.

**Comment by lafriks** [comment]
LGTM

**Comment by ethantkoenig** [comment]
LGTM, although these should eventually be moved to locales, right?

**Comment by lafriks** [comment]
@ethantkoenig I think yes

**Pull request merged**